libqf/libqfcore/src/core/utils.cpp

```diff
@@ -169,7 +169,43 @@ QString Utils::removeJsonComments(const QString &json_str)
 	// http://blog.ostermiller.org/find-comment
 	QString ret = json_str;
 	ret.replace(QRegularExpression("/\\*(?:.|[\\n])*?\\*/"), QString());
-	ret.replace(QRegularExpression("//.*[\\n]"), "\n");
```

Comment:
`ret.replace(QRegularExpression("//[^"]*[\\n]"), "\n");` nestacilo by tohle?

Reply:
To byl muj prvni pokus, ale nestacilo, pak to zase nezpracuje radek:
```
     //"port": ....
```
Pak jsem s temi regexpy jeste chvili valcil, neco jsem nasel, ale bylo to tak slozity, ze jsem se v tom absolutne nevyznal. A rikal jsem si ze nez tam davat neco cemu nerozumim a nemam nad tim zadnou kontrolu, tak ze radsi napisu par radek...

Reply:
a nemuzes tam dat, by to ignorovalo `://`?

Reply:
`(?<!:)//.*[\n]` bude fungovat

Reply:
pokud to podporuji QT

Reply:
neni to idealni reseni, ale za me lepsi, nez ta hromada radku

Reply:
nechapu treba, proc pocitas uvozovky za //, pokud je pred // lichy pocet uvozovek, muzes ho ignorovat, nic vic nepotrebujes

Reply:
> neni to idealni reseni, ale za me lepsi, nez ta hromada radku

Souhlas.

Reply:
Chodi to s tim.